### ISSUE TITLE
SWC style guide

### ISSUE THREAD

**Issue by swaldman3** [swcarpentry/communications#24]
As we start to talk about producing a number of different materials (e.g. brochures, annual reports, etc), it would be good to put together a style guide for consistency. At a minimum this should give,

* Fonts
* Colours
* What format to put the logo in (because you know somebody will make a JPG sooner or later and it will propagate)
...and so forth.

Doesn't have to be too prescriptive.

**Comment by iglpdc** [comment]
I agree. We have (a styles repo)[https://github.com/swcarpentry/styles] which we should somewhat keep in mind (e.g. not using a complete different set of colors).

There is also a version of the logo in these repo with some info about that, in particular the code for our SWC blue.

Regarding formats, svg (or something we can version control) is preferred to jpg.

**Comment by swaldman3** [comment]
Having seen tonight things as visually diverse as http://software-carpentry.org/badges/ and https://github.com/swcarpentry/site/pull/913/files, I think that a bit more than a simple list of fonts and colours is required. With multiple people working on SWC communications now, some decisions need to be made about the design language to be adopted across web and print media.

In my view this needs input from a competent graphic designer who can work with Greg and other stakeholders (perhaps including us here) and put together a brand/style guide. There may be somebody on board already who has the required skills and can lend the time, but if not, given the number of universities that are around here, perhaps a volunteer design student could be found?

**Comment by amyrbrown** [comment]
The question of getting a competent graphic designer is hard -- I don't think we have anyone on board already (although I may be wrong) and I'm worried about asking a design student to volunteer because I think we need to work with someone who has experience creating this kind of design package for a non-profit. That is, I don't think we have anyone on board who even knows what kinds of questions to ask or what to expect, so we need our graphic designer to tell us what we need (in much the same way that I'm likely to tell my clients what kind of editing they need, rather than ask them to tell me). We need someone to hold our hand.

So that brings it round to the usual question of funding versus finding volunteer hours -- do we want to pay for a design, or keep muddling on as we have, or try to beat the bushes some more for an existing contributor who can take this on? This might be a question for the Steering Committee.

Unless, of course, @swaldman3 would feel comfortable working with a novice designer! But we'd still have to find one, and figure out how to ask them to work for free. >_<

**Comment by swaldman3** [comment]
Hmm, I see the problem. And no, I fear that my expertise is not up to that. (also, I'm not heavily enough involved in SWC to feel comfortable being "client rep", IYSWIM).

Suggestion: Let's get something basic on fonts, colours, etc., up ASAP, and then try to make sure that pull requests that include significant visual things get reviewed from a design perspective as well as a purely functional one. "Does this fit with the existing look of things" as well as "does this do its job".

**Comment by amyrbrown** [comment]
It also seems that we should have an editorial style guide [like this](https://github.com/aosabook/500lines/wiki/Editing-Style-Guide), so poor @r-gaia-cs doesn't have to keep making comments [like this](https://github.com/swcarpentry/site/pull/907#discussion_r28199530) (or maybe the comments will still be required, but at least we'll have a record of preferences so we don't have to rely on the tastes of whoever happens to review a document)!

To my mind, an editorial style guide and a visual style guide should reside in the communications repo, rather than in the [styles](https://github.com/swcarpentry/styles) repo which seems to contain only CSS. I don't feel very strongly about that, though.

**Comment by r-gaia-cs** [comment]
To my mind, an editorial style guide and a visual style guide should reside in the communications repo, rather than in the styles repo which seems to contain only CSS.

+1.

**Comment by jduckles** [comment]
Worth reviving this discussion? Anyone want to put together a guide with where to get the logo, what colors to use etc?

**Comment by gvwilson** [comment]
No response since February, so closing.

**Comment by weaverbel** [comment]
Hi @gvwilson @jduckles  I have already written style guides for two places I have worked. I could possibly repurpose some of that work for this task - I'd have to check what is needed.

**Comment by tracykteal** [comment]
I'm also just starting to work with a graphics designer on a style guide. @weaverbel I'll coordinate with you.

**Comment by gvwilson** [comment]
Cool - who's the designer, and what's on their plate?
